adr/recipes/config_durations.py

```diff
@@ -35,6 +33,6 @@ def run(args, config):
         record.append(int(round(record[2] * record[3], 0)))
         result.append(record)
 
-    result = sorted(result, key=lambda k: k[args.sort_key], reverse=True)[:limit]
+    result = sorted(result, key=lambda k: k[args["sort_key"]], reverse=True)[:args["limit"]]
```

Comment:
We can keep `args` as a Namespace by doing this:

    from argparse import Namespace
    args = Namespace(**args)

That way we won't have to update all the recipes (plus I find the attribute lookup slightly nicer to work with).

Reply:
I modified the args so we can maintain the current way.